api/src/service/domain/workflow/workflowitem.ts

```diff
@@ -74,11 +74,11 @@ const schema = Joi.object().keys({
   createdAt: Joi.date().iso().required(),
   dueDate: Joi.date().iso().allow(""),
   displayName: Joi.string().required(),
-  // This should use exchangeRateSchema but can't, because of backward compatibility:
   exchangeRate: Joi.string()
     .when("amountType", {
       is: Joi.valid("N/A"),
       then: Joi.forbidden(),
+      break: true,
     })
```

Comment:
`break` was added to Joi in 16.x.x. This must be set to true, otherwise the next condition will be checked which is not the expected behavior.